pinc/User.inc

```diff
@@ -158,6 +163,8 @@ class User
         $userSettings =& Settings::get_Settings($this->username);
         if($action_type == "request")
             $userSettings->set_value("$activity$suffix", "requested");
+        elseif($action_type == "deny_request_for")
```

Comment:
We also seem to be missing handling for "revoke" (see public function revoke_access on line 141 of this module).

Reply:
"revoke" is handled as part of the else statement below.

Reply:
Ah. Right. I see that now.